### ISSUE TITLE
Unable to install Gnome Software due to older ubuntu-drivers-common version from Pop repo

### ISSUE THREAD

**Issue by quinzee33** [pop-os/beta#363]
**How did you upgrade to 21.10? (Fresh install / Upgrade)**

 
Fresh install
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

```
ubuntu-drivers-common:
  Installed: 1:0.9.4pop0~1648241952~22.04~031cbe1
  Candidate: 1:0.9.4pop0~1648241952~22.04~031cbe1
  Version table:
     1:0.9.6.1 500
        500 http://us.archive.ubuntu.com/ubuntu jammy/main amd64 Packages
 *** 1:0.9.4pop0~1648241952~22.04~031cbe1 1001
       1001 http://apt.pop-os.org/release jammy/main amd64 Packages
        100 /var/lib/dpkg/status
```

**Issue/Bug Description:**

Attempting to `sudo apt install gnome-software` results in the following error:

```
The following packages have unmet dependencies:
 software-properties-gtk : Depends: ubuntu-drivers-common (>= 1:0.9.6) but 1:0.9.4pop0~1648241952~22.04~031cbe1 is to be installed
E: Unable to correct problems, you have held broken packages.
```

**Steps to reproduce (if you know):**

Attempt to install Gnome Software on Pop OS 22.04

**Expected behavior:**

Gnome Software successfully being installed

**Other Notes:**

**Comment by n3m0-22** [comment]
https://github.com/pop-os/ubuntu-drivers-common/pull/22

**Comment by quinzee33** [comment]
This is also preventing the `nautilus-share` package for samba GUI controls in nautilus from being installed:

```
$ sudo apt install nautilus-share 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 software-properties-gtk : Depends: ubuntu-drivers-common (>= 1:0.9.6) but 1:0.9.4pop0~1648241952~22.04~031cbe1 is to be installed
E: Unable to correct problems, you have held broken packages.
```

**Comment by n3m0-22** [comment]
Fixed by [#22](https://github.com/pop-os/ubuntu-drivers-common/pull/22)
